src/main/java/edu/harvard/hul/ois/fits/tools/ToolBelt.java

```diff
@@ -171,9 +171,9 @@ private Tool createToolClassInstance(Class<?> toolClass, Fits fits) throws Refle
 			logger.debug("1-arg constructor for instantiating tool class: " + toolClass.getName());
 		} catch (NoSuchMethodException e) {
 			// now try a no-arg constructor
-			logger.warn("No Fits 1-arg constructor for tool class: " + toolClass.getName() + " -- trying no-arg constructor. Error message: " + e.getMessage());
+			logger.debug("No Fits 1-arg constructor for tool class: " + toolClass.getName() + " -- trying no-arg constructor. Error message: " + e.getMessage());

```

Comment:
While you're changing this log lines, it would be good to update them to the slf4j format:

```java
logger.debug("No Fits 1-arg constructor for tool class: {} -- trying no-arg constructor. Error message: {}", toolClass.getName(), e.getMessage());
```

This form is both safer and more efficient as the string concatenation only happens if the message is actually logged.

Reply:
Good point. Thanks!